### PR TITLE
fix: Previous submission link visibility, copy changes on landing, submission type pages

### PR DESCRIPTION
**Pull request by haworku**
## Summary
Previous submission link:
- See the `< ChangeHistory />` component
- Do not show link when only one version exists. When a new version has been resubmitted, show link

Copy changes:
- On the submission type page: Change “Programs” to “Programs this contract action covers” 
- On the Homepage: Remove “CHIP-Only Submissions” and “Documents for pre-review” from  the “Not accepted at this time” list. 

Cypress:
- Added new helpers `navigateFormByDirectLink` (formerly inline cy.visit scattered around),  `navigateFormByButtonClick` (formerly navigateForm) 
#### Related issues
https://qmacbis.atlassian.net/browse/MR-2465
https://qmacbis.atlassian.net/browse/MR-2422

#### Test cases covered
- 'should not include link to previous submission for the initial submission in the list'

## QA guidance
- Please do manual QA of the change history table in general. Including validating the submission versions linked are correct.

### CODE REVIEW

**Review by JasonLin0991 — CHANGES_REQUESTED** [review]
The copy changes look good. There seems to be a few issues on the change history work.

In the screen shot below. As a State or CMS user the most recent submission, when submission is not unlocked, should not have a link.
.

If we had multiple submissions and unlocks, the initial submission should have a link.
.

**Comment by JasonLin0991** [comment]
Forgot to mention in the review that the date time listed for submissions, does not match up with the date time when viewing previous submission.

File: services/app-web/src/components/ChangeHistory/ChangeHistory.tsx [hunk 1]
```diff
@@ -51,9 +51,15 @@ export const ChangeHistory = ({
...
         return result
     }
-    const revisedItems = flattenedRevisions().map((r) => {
+
+    const revisionHistory = flattenedRevisions()
+
+    const revisedItems = revisionHistory.map((r) => {
         const isInitialSubmission = r.updatedReason === 'Initial submission'
         const isSubsequentSubmission = r.kind === 'submit'
+        // We want to know if this package has multiple submissions. To have multiple submissions, there must be minimum
```
Comment (haworku) [inline services/app-web/src/components/ChangeHistory/ChangeHistory.tsx:60]: Thank you! this comment is clarifying lol :)

**Review by haworku — COMMENTED** [review]
🥇 Really like this change. Thanks for finding a clear way to call out how this link should display in the code. you will have to approve to merge

**Comment by rswerve** [comment]
Works as described and code looks good.

**Review by rswerve — APPROVED** [review]

**Review by JasonLin0991 — APPROVED** [review]
I also approve my changes.😎

**Pull request merged**